hbase-server/src/test/java/org/apache/hadoop/hbase/tool/TestLoadIncrementalHFilesByFamily.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.hadoop.hbase.tool;
+
+import org.apache.hadoop.hbase.HBaseClassTestRule;
+import org.apache.hadoop.hbase.HConstants;
+import org.apache.hadoop.hbase.codec.KeyValueCodecWithTags;
+import org.apache.hadoop.hbase.coprocessor.CoprocessorHost;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+
+public class TestLoadIncrementalHFilesByFamily extends TestLoadIncrementalHFiles {
```

Comment:
Does this test class actually make sense, without testing the specific new logic added? I mean, even without the group by family feature this test class will pass, so it's not shielding us from inadvertently changes on the new feature.

Reply:
yes, I delete this test , and add testBulkLoadByFamily ut .
bulkloadbyFamily only change client side , this config has no affect with server side . no need start another minicluster use different conf for test